utils/parser.py

```diff
@@ -67,8 +83,8 @@ def __init__(self, **kwargs: Any):
         WITHARG_CONST = ["while", "if", "elif"]
         self.WITHARG_REGEX = rf"(^(\s+)?(?P<captured>({self.form_re_const(WITHARG_CONST)})))(\s+).*((\s+)?:(\s+)?)"
 
-        self.UNCLOSED = rf".*(?P<unclosed>\()([^\)]+)?(?P<closed>\)?)"
-        self.CLOSED = rf".*(?P<closed>\))"
+        self.UNCLOSED = r".*(?P<unclosed>\()([^\)]+)?(?P<closed>\)?)"
+        # self.CLOSED = r".*(?P<closed>\))"
```

Comment:
can you please for fuck sake stop removing "unused" attributes when I never asked for it

Reply:
it is overwritten 2 lines below